### ISSUE TITLE
Add core releases and source files

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/localization#27]
The translation server needs a new language source file for every Backdrop release, and we have to make a translation release of it. We're *planning* to [generate the language files automatically](https://github.com/backdrop-ops/localize.backdropcms.org/issues/6). For the time being, I will generate the files and make respective translation releases, when a new minor Backdrop version is released.

**How to:**

- Add a release on the page `admin/l10n_server/projects/releases/backdropcms`.
- Upload the respective version of the translatable source file.
- Package the new release on `admin/l10n_server/packager`, and clear the "Page and else" cache.

**Releases:**

- [ ] 1.35 (to be published on September 15, 2026)
- [x] 1.34
- [x] 1.33
- [x] 1.32
- [x] 1.31
- [x] 1.30
- [x] 1.29
- [x] 1.28
- [x] 1.27
- [x] 1.26

Older releases, deleted because of display issues with too many releases:

- [ ] 1.25
- [ ] 1.24
- [ ] 1.23
- [ ] 1.22
- [ ] 1.21
- [ ] 1.20
- [ ] 1.19
- [ ] 1.18
- [ ] 1.17
- [ ] 1.16
- [ ] 1.15
- [ ] 1.14
- [ ] 1.13
- [ ] 1.12
- [ ] 1.11
- [ ] 1.10
- [ ] 1.9
- [ ] 1.8
- [ ] 1.7
- [ ] 1.6
- [ ] 1.5
- [ ] 1.4
- [ ] 1.3
- [ ] 1.2
- [ ] 1.1
- [ ] 1.0

**Comment by olafgrabienski** [comment]
Question: Does it make sense to add all releases?

**Comment by serundeputy** [comment]
I've added this repo: https://github.com/serundeputy/autogen_translation_releases

tl;dr
-----

A collection of scripts that are intended to run on cron on the localization server. they will
  * download backdrop latest release
  * download and run potx
  * create a release on the localization server corresponding the latest download
    * TODO: upload and parse the po file
  * there is a `cleanUp.sh` script to clean up the tmp things

details in the README.md

Perhaps we should move the repo to `backdrop-contrib` It is probably not useful for anyone but us, but in the name transparency it is probably the right place for it or maybe `backdrop-ops`.

**Comment by serundeputy** [comment]
I don't think we need all the old releases.

**Comment by jenlampton** [comment]
It also doesn't make sense to add a release for translations, for each bug-fix release of Backdrop. All the bug-fix releases of Backdrop are likely to have nearly the exact same strings. 

We should have a release for each minor version (1.1, 1.2, 1.3, etc) and I agree with @serundeputy that perhaps we should only show the 3 latest releases, since only 2 of those are officially "supported". 

I'm looking into the code in the packager module to see if there's a way we can limit the overview page (https://localize.backdropcms.org/translate/downloads) to the 3 latest releases, but also provide a way for people to get to older releases, if necessary.

**Comment by olafgrabienski** [comment]
> It also doesn't make sense to add a release for translations, for each bug-fix release of Backdrop. (...) We should have a release for each minor version (...) and I agree (...) that perhaps we should only show the 3 latest releases, but also provide a way for people to get to older releases, if necessary.

Agreed! I saw that the releases now have names like `1.13`, `1.12` and so on, nice! (Did you just rename the minor releases?)

**Comment by olafgrabienski** [comment]
When we [generate the language files automatically](https://github.com/backdrop-ops/localize.backdropcms.org/issues/6), I'd however prefer to add also bugfix releases, as there might be additional strings in such releases, and we won't have extra work to add them.

**Comment by jenlampton** [comment]
> Did you just rename the minor releases?

yep, you caught me ;)

> I'd however prefer to add also bugfix releases, as there might be additional strings in such releases, and we won't have extra work to add them.

That's what I was thinking too. But after we generate the files, we can add the updated strings for 1.13.2 to the 1.13 release :)

**Comment by klonos** [comment]
> ...after we generate the files, we can add the updated strings for 1.13.2 to the 1.13 release :)

Well, then how does one that has the 1.13 version of a translation know that there has been an updated version of it, that includes new strings? Should we be including a date/timestamp in their filename (the new regex in https://github.com/backdrop/backdrop-issues/issues/3974 should handle that)?

I think that long-term, we should have language translations be proper projects, with proper .info files, and proper versioning too. I think I would like to see something like a `backdrop-1.13-0.0.1.it.po` (for core), and `devel-1.7.0-0.0.1.it.po`. ...and I would ultimately like to see those handled by the Project Installer at some point too, so that people can install/update/remove translations via a familiar UI.

**Comment by jenlampton** [comment]
@klonos I love this vision!

**Comment by herbdool** [comment]
I've come to this while trying to port l10n_update and noticing the filenames don't match what is expected with its patterns matching. I'm not sure if it'll fallback to an earlier version or not. It's a complicated module. But hope that we can keep things stable here so we don't need to make any major changes to l10n_update (which is pretty useful for anyone trying to keep on top of new translations).

**Comment by olafgrabienski** [comment]
@herbdool Thanks for your message! Is already clear which filenames are supported `l10n_update`?

**Comment by olafgrabienski** [comment]
> I'm looking into the code in the packager module to see if there's a way we can limit the overview page (https://localize.backdropcms.org/translate/downloads) to the 3 latest releases, but also provide a way for people to get to older releases, if necessary.

@jenlampton In the meantime, the [Download page](https://localize.backdropcms.org/translate/downloads) is brimming with releases which even leads to display issues. Can you maybe have a look for a way to limit it to the latest releases?

**Comment by olafgrabienski** [comment]
With the 1.20 release, the Download page got even more crowded. I've deleted most of the older releases now, so that the list shows only the last four minor releases.

> In the meantime, the Download page is brimming with releases which even leads to display issues.
